website/content/docs/enterprise/lts.mdx

```diff
@@ -102,48 +101,36 @@ consider moving to a Vault Enterprise version with long-term support.
 
 ## What is long-term support?
 
-**Standard support** for Vault Enterprise includes:
+Long-term support offers extended maintenance (minor releases) for select, major
```

Comment:
Could you move minor releases out of the parens into the sentence like "Long-term support offers extended maintenance via minor releases ..." something like that

Reply:
fixed